php/register_account_details/register_account_details.php

```diff
@@ -0,0 +1,60 @@
+<?php
+include "../database.php";
+
+$conn = connect_local();
+
+header("Content-Type: application/json");
+
+$request = file_get_contents("php://input");
+
+if(isset($request) && !empty($request))
+{
+    $reqObj = json_decode($request);
+    //Checking for duplicates depending on input
+    switch($reqObj->method)
+    {
+        case "check_duplicates_email":
+            $user = get_user_by_mail($conn, $reqObj->email);
+            if($user == false){
+                echo json_encode(["error" => false, "duplicate" => false]);
+            }
+            else if($user != false){
+                echo json_encode(["error" => false, "duplicate" => true]);
+            }
+            else if($user == null){
```

Comment:
return $user == null ? json_encode(["error" => false, "duplicate" => false]) : json_encode(["error" => true, "duplicate" => true]);
Perhaps error true or false and not duplicate but an appropriate message (for the toast)?